sf-perftest-testapp/README.md

```diff
@@ -0,0 +1,36 @@
+# stormforger-testapp
+
+Simple experiments to optimize the SF PerfTest [testapp](https://github.com/stormforger/testapp/) via Optimize Pro.
+NOTE: This is not really useful as an optimization target since the testapp doesn't really have a lot of knobs to tune nor needs a lot of memory/cpu. 
+However, it is useful as a test target for experimenting with Opt Pro trial pods for various performance testing tools.
+
+Deploy the testapp to your kubernetes:
+
+```sh
+kubectl apply -f ./testapp.yaml
+```
+
+If you need an Amazon Load Balancer ingress, e.g. for your `labs` cluster:
+
+```sh
+kubectl apply -f ./alb-ingress.yaml
+```
+
+If you want to use Prometheus metrics from your trial pods using the Prometheus setup task in your experiment,
+you will need some RBAC settings:
+
+```sh
+kubectl apply -f stormforge-setup-rbac.yaml
```

Comment:
I decided to move the RBAC config into each `experiment.yaml` because that seems to be the convention used elsewhere (that's what's generated by `stormforge run` for example).